### ISSUE TITLE
Excessive log generation by kubelet v1.12.2

### ISSUE THREAD

**Issue by gree-gorey** [kubernetes/kubernetes#71031]
**What happened**:
Because of the bug #69015 `kubelet` writes too many logs to syslog, producing ~5G of them per day which is filling up the rootfs very fast (and makes the whole cluster unhealthy; e.g. I have 10G of rootfs).

**What you expected to happen**:
`kubelet` produces reasonable amount of logs by default, or at least having a way to decrease this verbosity.

**How to reproduce it (as minimally and precisely as possible)**:
Upgrade `kubelet` to `1.12.2` and watch the size of `/var/log/syslog*` files every day.

**Anything else we need to know?**:

**Environment**:
- Kubernetes version (use `kubectl version`):
```
Server Version: version.Info{Major:"1", Minor:"12", GitVersion:"v1.12.2", GitCommit:"17c77c7898218073f14c8d573582e8d2313dc740", GitTreeState:"clean", BuildDate:"2018-10-24T06:43:59Z", GoVersion:"go1.10.4", Compiler:"gc", Platform:"linux/amd64"}
```
- Cloud provider or hardware configuration: AWS
- OS (e.g. from /etc/os-release): 
```
NAME="Ubuntu"
VERSION="16.04.5 LTS (Xenial Xerus)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 16.04.5 LTS"
VERSION_ID="16.04"
HOME_URL="http://www.ubuntu.com/"
SUPPORT_URL="http://help.ubuntu.com/"
BUG_REPORT_URL="http://bugs.launchpad.net/ubuntu/"
VERSION_CODENAME=xenial
UBUNTU_CODENAME=xenial
```
- Kernel (e.g. `uname -a`):
```
Linux 4.4.0-1057-aws #66-Ubuntu SMP Thu May 3 12:49:47 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux
```
- Install tools: `kubeadm 1.12.2`
- Others:

/kind bug

**Comment by dims** [comment]
/milestone v1.13 

Hoping https://github.com/kubernetes/kubernetes/pull/70494 for https://github.com/kubernetes/kubernetes/issues/69015 will fix this issue too!

**Comment by saad-ali** [comment]
Do you have sample of the log spam? That will help us figure out the culprit component.

**Comment by dims** [comment]
@saad-ali there's an example in https://github.com/kubernetes/kubernetes/issues/69015

**Comment by gree-gorey** [comment]
that's right, I have the same output, most of which are `error dial failed at socket` and `error could not find plugin for deleted file`, producing several lines a second indefinitely

**Comment by AishSundar** [comment]
@saad-ali @dims @gree-gorey while this is something introduced in 1.13, how critical is it to be fixed in 1.13 vs a patch release? the reason I ask is, code freeze is less than a day away and at this point we are hoping #70494 would fix it. In the event of it not addressing this issue are we ok living with it until 1.13.1?

**Comment by RenaudWasTaken** [comment]
Closed by https://github.com/kubernetes/kubernetes/pull/70494
